test/staking/SwapHelper.test.ts

```diff
@@ -0,0 +1,499 @@
+import { ethers, upgrades } from "hardhat";
```

Comment:
1. if you copy paste , copy from my latest tests
2. remove all unused variables and imports
3. all unrequire things in beforeAll

Reply:
Removed unusued variables and imports
Removed unrequired things from beforeAll